src/main/java/cafe/Cafe.java

```diff
@@ -0,0 +1,48 @@
+package cafe;
+
+import coupon.Coupon;
+import io.output.Output;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Cafe {
+    private static final int FIRST_ORDER = 1;
+    private static final int ONE = 1;
+    private BigDecimal sales;
+    private Map<Integer, Order> orderHistory;
+    private Map<Coupon, Integer> couponHistory;
```

Comment:
```suggestion
    private final Map<Integer, Order> orderHistory;
    private final Map<Coupon, Integer> couponHistory;
```

필드는 생성자에서 초기화되니 final을 붙여주시는 것이 좋습니다.!

Reply:
넵! 수정하겠습니다.
감사합니다.